src/pages/about-us/team/index.tsx

```diff
@@ -1,13 +1,45 @@
 // path: about-us/team/leadership
 
 import { Typography } from '@mui/material';
+import { GetServerSideProps } from 'next';
 
-const TeamLeadershipPage = () => {
+// import { TeamCoreData, TeamContactData, TeamMembersData } from '@utils/types';
+import { TeamApiResponse } from '@utils/types';
+import { fetchData } from 'lib/api';
+
+interface TeamPageProps {
+  team: TeamApiResponse;
+}
+
+const TeamLeadershipPage = ({ team }: TeamPageProps) => {
+  const { membersByType } = team;
   return (
     <div>
+      {membersByType.directors.map((director) => (
+        <span key={director.fullName}>{director.fullName}</span> //Example how to use the data
+      ))}
       <Typography variant="h4">Welcome to the TeamPage</Typography>
```

Comment:
Should this be: page.title or page.subtitle? 

Also it is missing: page.description.

```
"page": {
    "title": "Team",
    "subtitle": "Core Team",
    "description": "The core team of our community is composed of visionary directors and dynamic leaders, who bring a wealth of experience and dedication to empowering women in technology. These esteemed professionals are committed to fostering an environment of growth, inclusivity and support, guiding our community toward achieving its mission and goals."
  }
```

Reply:
This is just temporary to show the connection between the API and the frontend, I could just do data.json to display all. It will be removed when the components are ready.